src/spans/awsSpan.js

```diff
@@ -109,8 +121,9 @@ export const getFunctionSpan = () => {
   const started = new Date().getTime();
   const ended = started; // Indicates a StartSpan.
 
-  const event = payloadStringify(parseEvent(lambdaEvent));
-  const envs = payloadStringify(process.env);
+  // We need to keep sending them in the startSpan because we don't always have an endSpan
+  addEventToSpan(basicSpan);
+  addEnvsToSpan(basicSpan);
```

Comment:
It's better than its build-in 1 step, we will makespan frozen soon after we will add typescript, so please keep this at the way it was

Reply:
Just take care of error stuff